docs/deploying.md

```diff
@@ -444,3 +444,36 @@ inventory for each resource.
 Each service specific condition will be set to `True` as that service completes
 successfully. Looking at the service conditions will indicate which services
 have completed their deployment, or in failure cases, which services failed.
+
+### OpenStackDataPlaneNodeSet deployment hashes
+
+Each `OpenStackDataPlaneService` can optionally have an associated list of
+`ConfigMaps` and `Secrets` that are mounted as file data into the
+`OpenStackAnsibleEE` job started to deploy that service. The ansible content
+then is able to consume those files as necessary. See [Configuring a custom
+service](composable_services.md#configuring-a-custom-service) for more details.
+
+When an `OpenStackDataPlaneDeployment` succeeds, the computed hash of each
+`ConfigMap` and `Secret` for each `OpenStackDataPlaneService` that was deployed
+is saved on the status of each `OpenStackDataPlaneNodeSet` referenced by the
+`OpenStackDataPlaneDeployment`.
+
+For example, the following hashes are saved on the `OpenStackDataPlaneNodeSet`
+status after a typical deployment:
+
+```console
+$ oc get openstackdataplanenodeset openstack-edpm -o yaml
+
+<snip>
+status:
+  conditions:
+	<snip>
+  configMapHashes:
+    ovncontroller-config: n655h5...
+  secretHashes:
+    neutron-dhcp-agent-neutron-config: n9ch5...
+    neutron-ovn-metadata-agent-neutron-config: n588h...
+    neutron-sriov-agent-neutron-config: n648h...
+    nova-cell1-compute-config: n576h...
+    nova-metadata-neutron-config: n56fh...
+```
```

Comment:
I would add that:
These hashes can be compared against the hash of the CM / Secret to see if there is newer input data that has not been deployed to NodeSet. For example if the hash of `nova-cell1-compute-config` in the NodeSet Status is differnt from the hash of `nova-cell1-compute-config` in the NovaCell/cell1 status. Then there is nova-compute control plane configuration data the needs to be deployed to the EDPM compute nodes.